en/CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing guidelines for Chinese-to-English
+
+We are working on translating "Hello Algo" from Chinese to English with the following approach:
+
+1. **AI translation**: Carry out an initial pass of translations using the machine learning translator.
+2. **Human optimization**: Manually refine the machine-generated outputs to ensure authenticity and accuracy.
+3. **Pull request review**: The optimized translation will be doubly checked by the reviewers through GitHub pull request workflow.
+4. Repeat steps `2.` and `3.` for further improvements.
+
+## Join us
+
+We're seeking contributors who meet the following criteria.
+
+- **Technical background**: Strong foundation in computer science, particularly in data structures and algorithms.
+- **Language skills**: Native proficiency in Chinese with professional-level English, or native English.
+- **Available time**: Dedicated to contributing to open-source projects with a willingness to engage in long-term translation efforts.
+
+That is, our contributors are computer scientists, engineers, and students from different linguistic backgrounds, and their objectives have different focal points:
+
+- **Native Chinese with professional working English**: Ensuring translation accuracy and consistency between CN and EN versions.
+- **Native English**: Enhance the authenticity and fluency of the English content to read natural and engaging.
```

Comment:
'natural' needs to be conjugated. I suggest "Enhance the authenticity and fluency of the English content to read naturally and to be engaging"